### DISCUSSION TITLE
Do it work good on Windows Server?

### DISCUSSION THREAD

**Discussion by biapar** [strapi/strapi#7391] [Legacy Topics]
Do it work good on Windows Server?

**Comment by derrickmehaffy** [comment]
Yes Strapi will work on a windows server, though I would advise using a Linux server in production unless you have a specific reason why you can't.

**Reply by biapar** [reply]
I've more familiarity with WinServer. Are there some constraints or issues to run on it?

**Reply by derrickmehaffy** [reply]
It will work but is strongly recommended against, also when it comes to using a web-server for proxying (SSL, ect) I would strongly advise against using IIS and instead use something like Nginx.

In terms of cost and performance, Windows server will generally cost quite a bit more and their performance (due to how the file system is managed) is much slower.

**Reply by biapar** [reply]
Thanks.
